action.yml

```diff
@@ -5,6 +5,11 @@ branding:
   icon: 'check-circle'
   color: 'green'
 inputs:
+  repo:
+    description: 'the target `owner/repo` to manage the check run on (defaults to the current repository'
+    required: false
+  sha:
+    description: 'the target commit''s SHA (defaults to the current commit)'
```

Comment:
Does it need to have `required: false` explicitly like the other?

Reply:
It doesn't ([false is default](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_idrequired)), but I've added it for consistency - missed it before.